setup.py

```diff
@@ -52,11 +52,13 @@ def find_version(*file_paths):
         'Natural Language :: English',
         'License :: OSI Approved :: Apache Software License',
         'Programming Language :: Python',
+        "Programming Language :: Python :: 3 :: Only",
```

Comment:
Just curious why add this now?

Reply:
It was missed previously.

Reply:
This isn't included in any of the other repos we own. If we add it here, we should add it to the others as well.